spec/lib/fetcher_service_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "fetcher/fetcher_service"
+require "fetcher/fetcher"
+
+RSpec.describe FetcherService do
+  describe "#fetch" do
+    it "delegates to a configured fetcher" do
+      fetcher = instance_double("v1")
+      allow(fetcher).to receive(:fetch).with(any_args)
+      fetcher_service = described_class.new([fetcher])
+      expect(fetcher).to receive(:fetch).with("https://www.example.com")
+      fetcher_service.fetch("https://www.example.com")
+    end
+
+    it "only goes to fetcher_two if fetcher_one raises an error" do
+      fetcher_one = instance_double("v1")
```

Comment:
Any reason we've used `one` over `1`?

```suggestion
      fetcher1 = instance_double("v1")
```